atum/src/main/scala/za/co/absa/atum/core/ControlFrameworkState.scala

```diff
@@ -39,6 +40,7 @@ class ControlFrameworkState(sparkSession: SparkSession) {
   private[atum] var workflowName: String = ""
   private[atum] var havePendingCheckpoints: Boolean = false
   private[atum] var allowUnpersistOldDatasets: Boolean = false
+  private[atum] var cacheStorageLevel: Option[StorageLevel] = Some(StorageLevel.MEMORY_AND_DISK)
```

Comment:
If you have a nice default there, why make it an option? There is an actual `StorageLevel.NONE`. More in [docs](https://spark.apache.org/docs/2.2.1/api/java/org/apache/spark/storage/StorageLevel.html#NONE--)

Reply:
The reason is that if caching is disabled I'd like to skip calls to `.persist()` at all, and not do `.persist(StorageLevel.NONE)`.
True, these 2 things probably achieve the same. But when caching is disabled I wanted to leave it no chances. 😄